lightning/src/offers/refund.rs

```diff
@@ -248,8 +248,11 @@ macro_rules! refund_builder_methods { (
 	///
 	/// Successive calls to this method will add another blinded path. Caller is responsible for not
 	/// adding duplicate paths.
-	pub fn path($($self_mut)* $self: $self_type, path: BlindedPath) -> $return_type {
-		$self.refund.paths.get_or_insert_with(Vec::new).push(path);
+	pub fn path($($self_mut)* $self: $self_type, paths: Vec<BlindedPath>) -> $return_type {
+		let entry = $self.refund.paths.get_or_insert_with(Vec::new);
+		for path in paths {
+			entry.push(path);
+		}
```

Comment:
Likewise.

Reply:
And Done! [f32a93a](https://github.com/lightningdevkit/rust-lightning/commit/f32a93acc60d20078c42b96dbb4466969b06f5b3)
Thanks! :)